app/javascript/react/components/Board.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import Square from "./Square"
+
+const Board = (props) => {
+  const [selectedSquare, setSelectedSquare] = useState(null)
+
+  let boardData = []
+  let letters = ["h", "g", "f", "e", "d", "c", "b", "a"]
+  for (let i = 1; i <= 8; i++) {
+    letters.forEach((letter) => {
+      boardData.unshift(
+        <Square
+          row={i}
+          column={letter}
+          selectedSquare={selectedSquare}
+          setSelectedSquare={setSelectedSquare}
+        />
+      )
+      if (letter === "a") {
+        boardData.unshift(
+          <br/>
+        )
+      }
+    })
+  }
+
+  return <div id="board">{boardData}</div>
+}
+
```

Comment:
looks fine, but I would avoid relying on that `br` tag for styling. ESPECIALLY because your Board data is supposed to be made up of squares, not squares and html elements